blog/2023-07-27-introducing-substratus.md

```diff
@@ -0,0 +1,137 @@
+  plan on uninstalling Flight Simulator tonight...✈️
+* Is any part of this workflow portable or repeatable? ♻️
+
+It would be reasonable to walk away from a night of yak-shaving with a Colab
+subscription and a handful of modest wins. But your plan to deliver a secure,
+consistent, and portable ML workflow to your day job is still effectively
+🤷🏾‍♀️. Even through great effort and ample research, your ultimate strategy
+might be: **hope**, that your engineering team is satisfied with Sagemaker's
+technical choices and your CFO doesn't care about pricing. 💸
+
+There's got to be a better way.
+
+## Why Substratus?
+
+**Avoid vendor lock-in**: Machine learning workloads operate on undifferentiated
+compute primitives, abundant across all cloud platforms (CPU, GPU, storage,
+networks). GPUs may be less-available but are no-less commodity than the rest of
+the ML hardware stack. It takes *some* legwork to orchestrate these components
+to be useful in an ML context, but adopting a platform shouldn't come at the
+cost of lock-in.
+
+**Keep company data internal**: There's a tension playing out in real-time
+across organizations today. AI-assisted workers report both greater satisfaction
+in their jobs and non-trivial productivity gains. These trends will only
+accelerate. This should be a win-win for companies and employees alike, but
+there's a rub - the best models are hosted by another company - potentially a
+direct competitor. But even if they aren't a direct competitor, your company
+secrets are absolutely at risk. Operating your own internal inferencing services
+is the most secure path to realizing the business opportunity AI presents.
+
+**Rapid experimentation with the latest ML models**: Substratus makes it easy
+for developers to fine-tune, serve, and experiment with open-source ML models in
+notebooks. This can be done in minutes not hours using our off-the-shelf
+loaders, trainers, and servers. All this is possible without writing any code.
+
+**Best practices baked-into your ML artifacts**: While Substratus is
+less-opinionated than other platforms about your team's exact workflow, we
+**are** rather opinionated about keeping core parts of the development process
+on rails. Once created, Models, Datasets, and Servers are all immutable. They
+can spawn new objects of the same kind (think: fine-tuning a model), but they
+can't be changed. Similarly, our notebook environments are containerized and
+deployable to prod. Finally, we're working on canned model optimizations like
+auto-quanitization to enabling efficient serving whenever possible.
+
+**Open-core**: Substratus will always be open-core. Period. If your team can
+support it - and we believe many can - deploy the platform in as many places as
+you need and at whatever scale suits the business.
+
+While there are other open-source projects in this space, we've taken careful
+notes on how we want to differentiate.
+
+## What are our opinions as we launch?
```

Comment:
Are there any other design north stars we should highlight? To me, this answers the question of how we differ from the likes of kubeflow